.github/workflows/enforce-license-compliance.yml

```diff
@@ -2,15 +2,19 @@ name: Enforce License Compliance
 
 on:
   push:
-    branches: [master, release/*]
+    branches:
+      - master
+      - release/**
+      - release-library/**
```

Comment:
Remind me what the difference is between one and two stars here? 🐭

Reply:
Two stars includes further slashes.

Reply:
That's right, I went back and forth and went for the basic option, probably doesn't matter too much either way.

Reply:
(Unless you know that it does. :)

Reply:
Ah, that makes sense. 😁